### ISSUE TITLE
Question on output of SRL model

### ISSUE THREAD

**Issue by panwarnaveen9** [allenai/allennlp#4467]
```python
from allennlp.predictors.predictor import Predictor
import spacy

predictor = Predictor.from_path("https://storage.googleapis.com/allennlp-public-models/bert-base-srl-2020.03.24.tar.gz")

sentence = "Apple corporation sold 1 million computers this month"

nlp = spacy.load("en_core_web_sm")
doc = nlp(sentence)

words = sentence.split()
verb_labels = [0 for _ in words]
verb_labels[2] = 1

instance = predictor._dataset_reader.text_to_instance(doc, verb_labels)
output = predictor._model.forward_on_instance(instance)

print("logits", output["logits"].shape)
print("probabilities", output["class_probabilities"].shape)
```

it returned me a shape of numpy array as follow

```
(12, 130)
(12, 130)
```

How can I use understand this ? 
Sequence length is 8 and what is 130 ?

**Comment by dirkgr** [comment]
I don't know the SRL model exactly, but I would guess there are 130 different classes that a token can be in, and each value gives you the probability that it's that class. `"probabilities"` is probably the softmax over `"logits"`, so they contain the same information.

**Comment by panwarnaveen9** [comment]
Thanks for comment

I guessed same thing.. but where can I get the list of these SRL classes so that I can map each value to some class label. 
Second question was regarding value `12` as `shape[0]` which is not directly map to sequence length - `8`

**Comment by dirkgr** [comment]
BERT re-tokenizes the given string, and it adds special tokens. So in your case, you have 8 input tokens, but BERT adds a `[CLS]` token, a `[SEP]` token, and it probably uses word-pieces for words like "corporation" or "computers", for a total of 12 tokens in the output.

I don't know what the 130 classes are. You can probably find that information in the Ontonotes 5.0 dataset documentation.
